lang/python/pytest/patches/001-do-not-use-setuptools-scm.patch

```diff
@@ -0,0 +1,21 @@
+Index: pytest-5.0.0/setup.py
+===================================================================
+--- pytest-5.0.0.orig/setup.py
++++ pytest-5.0.0/setup.py
```

Comment:
Would you please remove it the exact version? If there is a new release, you will need to rebase the patch. There's a chance that you will don't need to do it for each version. In that case, the refresh should be enough.

Reply:
Removed